CompanyEmployees.Core.Domain/Entities/Attendance.cs

```diff
@@ -0,0 +1,29 @@
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace CompanyEmployees.Core.Domain.Entities
+{
+    public class Attendance
+    {
+        [Column("AttendanceId")]
+        public Guid Id { get; set; }
+
+        [Required]
+        [ForeignKey(nameof(Employee))]
+        public Guid EmployeeId { get; set; }
+
+        [Required]
+        public DateTime CheckInTime { get; set; }
+
+        public DateTime? CheckOutTime { get; set; }
+
+        [Required]
+        [MaxLength(10, ErrorMessage = "Maximum length for the Status is 10 characters.")]
```

Comment:
Consider removing the nullable annotation (`?`) from the `Status` property since it's marked as `[Required]`, which suggests the value should never be null.

**Suggested change**
```suggestion
public string Status { get; set; }
```